NYC_Summit18_Developer_Workshop/workspace/.metadata/.plugins/org.eclipse.rse.core/profiles/PRF.ip-172-31-29-133_0/FP.local.files_0/node.properties

```diff
@@ -1,5 +1,5 @@
 # RSE DOM Node
-00-name=ip-172-31-90-94\:local.files
+00-name=ip-172-31-29-133\:local.files
```

Comment:
Are these added per node? Do we need to change them to not have your instance ip in it?

Reply:
These files are created as part of the .metadata.  If we do not include the .metadata directory then the pre-configured SDAccel GUI doesn't come out in right configuration. Can't delineate which .metadata files are required for the SDAccel GUI, hence had to check-in the full .metadata directory.

Reply:
Can you verify that you can clone this branch on a different machine and still be able to bring up GUI.

Also check what happens when this PRF file doesn't exist.. That way we can get rid of information like this

Reply:
Already verified the GUI will work after cloning on a different instance.